### PR TITLE
feat: Lower storage cost 10X

### PR DESCRIPTION
**Pull request by evgenykuzyakov**
Introduces a protocol change to lower storage cost 10X, to 1e19 per byte (current cost is 1e20 per byte).

This introduces a stable protocol feature instead of nightly, since we want to release it directly to testnet.

Gov forum discussion: https://gov.near.org/t/storage-staking-price/399

## Test plan:

- [x] CI
- [x] nayduck http://nayduck.eastus.cloudapp.azure.com:3000/#/run/1082
- [X] unit_test

### CODE REVIEW

File: core/runtime-configs/src/lib.rs [hunk 1]
```diff
@@ -120,4 +148,18 @@ mod tests {
...
+    fn test_lower_cost() {
+        let config = Arc::new(RuntimeConfig::default());
+        let config_same = RuntimeConfig::from_protocol_version(&config, 0);
+        assert_eq!(
+            config_same.as_ref().storage_amount_per_byte,
+            config.as_ref().storage_amount_per_byte
+        );
+        let config_lower = RuntimeConfig::from_protocol_version(&config, ProtocolVersion::MAX);
+        assert!(
+            config_lower.as_ref().storage_amount_per_byte < config.as_ref().storage_amount_per_byte
```
Comment (bowenwang1996) [inline core/runtime-configs/src/lib.rs:162]: can we assert that one is 10x of the other

**Review by bowenwang1996 — APPROVED** [review]
I thought we planned to reduce it by 20x. @ilblackdragon

File: core/runtime-configs/src/lib.rs (hunk 1 above)
Comment (evgenykuzyakov) [reply core/runtime-configs/src/lib.rs:162]: Default is different from mainnet genesis

**Comment by olonho** [comment]
BTW, what is the motivation behind this change?

**Comment by evgenykuzyakov** [comment]
> BTW, what is the motivation behind this change?

@olonho https://gov.near.org/t/storage-staking-price/399 and linked video discussion https://www.youtube.com/watch?v=Ge1Cfq-0y5w

**Comment by ilblackdragon** [comment]
I would vote to go with 20x reduction as we discussed during the call.

**Comment by evgenykuzyakov** [comment]
@ilblackdragon  @bowenwang1996 I didn't agree to 20X on the call. We should do official voting process to discuss 10X vs 20X. We also need to do this asap if we still want to ship it.

**Comment by evgenykuzyakov** [comment]
After discussion on the forum and in NEAR Dev telegram group, we've decided to keep 10X reduction as implemented in this PR. It allows us to move to 20X reduction later when we need it.

**Pull request merged**